operations/sthree.go

```diff
@@ -307,7 +307,11 @@ func s3SyncToCmd() cli.Command {
 					DryRun:       c.Bool("dry-run"),
 					DeleteOnSync: c.Bool("delete"),
```

Comment:
do we want to add `DeleteOnPull` and `DeleteOnPush` options?

Reply:
for the CLI to do both, I'm not eager to add the new options here, right now.